gaia-media-controls.js

```diff
@@ -0,0 +1,291 @@
+    top: calc(50% - 0.1rem); /* 1 */
+    z-index: 20; /* 2 */
+  }
+
+  .time-background {
+    width: 100%;
+    height: 0.1rem;
+    top: calc(50% - 0.3rem); /* 1 */
+    background-color: #a6b4b6;
+    z-index: 10; /* 2 */
+  }
+  /*
+   * 1. Center 'play-head' vertically. 'top' is relative to the other
+   *    'slider-wrapper' elements which are 0.4rem in height; therefore,
+   *    'top:50%' would position the top of the element 0.4rem below the
+   *    the center of the 'sider-wrapper'. In order to position the center
+   *    of the element vertically, move it up by half its height plus
+   *    the height of the previously layed out elements.
+   *
+   * 2. Ensure the layering order of time background,
+   *    elapsed time, and play head.
+   */
+  .play-head {
+    top: calc(50% - (1.15rem + 0.4rem));
+    position: relative;
+    width: 2.3rem;
+    height: 2.3rem;
+
+    /* For LTR langauges, position the playhead 1.15 rems to the left
+     * so that the center of the playhead aligns with the beginning of
+     * the time slider.
+     */
+    margin-left: -1.15rem;
+
+    /* For RTL langauges, position the playhead 1.15 rems to the right
+     * so that the center of the playhead aligns with the end of
+     * the time slider.
+     */
+    margin-right: -1.15rem;
+
+    border: none;
+    background: none;
+    pointer-events: none;
+    z-index: 30; /* 2 */
+  }
+
+  /*
+   * Define the 'normal' play-head graphic. Using the 'after' pseudo-element
+   * here specifies that the 'normal' (smaller, white) play-head will
+   * appear on top of the larger, blue 'active' play-head (specified using
+   * the 'before' pseudo-element).
+   */
+  .play-head:after {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: calc(50% - 1.15rem);
+    width: 2.3rem;
+    height: 2.3rem;
+    border-radius: 50%;
+    background-color: #fff;
+  }
+
+  /* Define the 'active' play-head graphic (blue, larger than the 'normal'
+   * play-head). Using the 'before' pseudo-element specifies that the 'active'
+   * play-head will appear under the 'normal' play-head.
+   */
+  .play-head.active:before {
+    content: "";
+    position: absolute;
+    top: calc(50% - 3.05rem);
+    left: calc(50% - 3.05rem);
+    width: 6.1rem;
+    height: 6.1rem;
+    border-radius: 50%;
+    background-color: #00CAF2;
+  }
+
+  /* video control bar -- rewind, pause/play, forward
+   *
+   * 1. The buttons should always display left-to-right.
+   */
+  .video-control-bar {
+    display: flex;
+    flex-direction: row;
+    flex-basis: 4.8rem;
+    border-top: 0.1rem solid rgba(255,255,255, 0.1);
+    background-color: rgba(0,0,0, 0.95);
+    overflow: hidden;
+    direction: ltr; /* 1 */
+    /*z-index: 10*/;
+  }
+
+  .seek-backward,
+  .seek-forward,
+  .play {
+    /* All three elements grow and shrink together by the same proportion */
+    flex-grow: 1;
+    padding: 0;
+  }
+
+  .play.paused:before {
+    content: 'play';
+    padding-left: 4px;
```

Comment:
I don't understand why the padding-left is required when we're paused but not when we're playing. If you need a hardcoded constant, it should be 0.4rem, not 4px.  But you don't have padding for the other buttons, so I don't see why you need it here. 

If you really can't remove this, add a comment, at least.